src/controllers/connectionManager.ts

```diff
@@ -856,8 +858,31 @@ export default class ConnectionManager {
 				connectionInfo.connectHandler = ((connectResult, error) => {
 					if (error) {
 						reject(error);
+						// TODO: add scrubbed error details to telemetry
```

Comment:
Make sure you're making an issue to track this (and probably just remove the comment...TODO comments are generally pretty pointless for actually tracking work)

Reply:
Shouldn't you be adding this to handleConnectionCompleteNotification where error code is available unless I'm missing something?

Reply:
Added. Thanks for pointing it out